backend/service/redis.go

```diff
@@ -0,0 +1,48 @@
+package service
+
+import (
+	"context"
+	"time"
+
+	"github.com/go-redis/redis/v8"
+)
+
+type RedisIFace interface {
+	Set(ctx context.Context, key string, value interface{}, expiration time.Duration) error
```

Comment:
what are the exact values that could be passed into redis? In the [redis docs](https://redis.io/docs/data-types/) it seems like the primitive type is a "string," which seems like the only thing we'll ever store.

Reply:
The idea was to follow the same function signature that the redis library uses: 

```
func (redis.cmdable).Set(ctx context.Context, key string, value interface{}, expiration time.Duration) *redis.StatusCmd
```